lib/integrations/drivers/intercom.rb

```diff
@@ -0,0 +1,60 @@
+### Intercom integration
+# - the integration we use to facilitate comunication in the app
+# and get users to corelate
+module Integrations::Drivers
+  class Intercom < Integrations::BaseDriver
+    def self.display_name
+      'Intercom'
+    end
+
+    def type
+      :intercom
+    end
+
+    def list_subscribers
+      config = self.configuration
+      users = []
+      resource = RestClient::Resource.new( api_url + 'users', config['token'], nil)
+      response = resource.get( :content_type => :json, :accept => :json)
```

Comment:
try to make a get method as we might be using the same calls later in other methods.